__main__.py

```diff
@@ -23,6 +24,25 @@ def __init__(self, window):
         self.enemy_list: Optional[arcade.SpriteList] = None
         self.player: Optional[Player] = None
 
+        self.music_list = []
+        self.song_index = 0
+        self.current_player = None
+        self.music = None
+
+    def advance_song(self):
+        """
+        Advance our pointer to the next song. This does NOT play the song.
+        :return:
+        """
+        self.song_index += (self.song_index + 1) % len(self.music_list)
+
+    def play_song(self):
+        self.music = arcade.Sound(self.music_list[self.song_index], streaming=True)
+        self.current_player = self.music.play(
+            0.1
+        )  # should use config.CONFIG.music_volume when ui-elements is merged
+        time.sleep(0.03)
+
     def setup(self):
```

Comment:
Can we have the music / sound contained within a class so it can be easily deconstructed and remade using `setup()` for level transitions etc. 

Additionally it should be cleaner as all our Audio/Music methods and logic will be contained there. Rather than cluttering the MainView

Reply:
good idea, wilco